social_core/backends/google_onetap.py

```diff
@@ -0,0 +1,53 @@
+import urllib3
+from google.auth.transport.urllib3 import Request
+from google.oauth2 import id_token
+
+from social_core.backends.base import BaseAuth
+from social_core.backends.google import BaseGoogleAuth
+from social_core.exceptions import AuthException, AuthTokenError
+
+
+class GoogleOneTap(BaseGoogleAuth, BaseAuth):
+    name = "google-onetap"
+    CSRF_KEY = "g_csrf_token"
+    CREDENTIAL_KEY = "credential"
+
+    def auth_url(self):
+        raise AuthException(self, "Cannot start login flow for Google One Tap")
+
+    def verify_csrf(self, request):
+        csrf_token_body = self.data.get(self.CSRF_KEY)
+        csrf_token_cookie = request.COOKIES.get(self.CSRF_KEY)
+
+        if not csrf_token_body:
+            raise AuthTokenError(self, "Missing csrf token from response")
+
+        if not csrf_token_cookie:
+            # csrf_token_cookie can be missing due to
+            # https://issuetracker.google.com/issues/226157137
+            return
```

Comment:
This makes the whole CSRF verification mostly useless as it does not verify requests without cookies.

Reply:
I agree, I was thinking of making it configurable so users who have everything on a single domain have a sane default and users with who have the auth part on a different one can disable it, I'll add that with the tests.